akka-http-tests/src/test/java/docs/http/javadsl/server/HandlerExampleDocTest.java

```diff
@@ -30,16 +30,16 @@ public void testSimpleHandler() {
       //#simple-handler
       Route handlerString = extractMethod(method ->
         extractUri(uri ->
-          complete(String.format("This was a %s request to %s", method, uri))
+          complete(String.format("This was a %s request to %s", method.name(), uri))
```

Comment:
Currently toString is "HttpMethod(GET)". Not sure if it is a good idea having it toString to the name. You decide.